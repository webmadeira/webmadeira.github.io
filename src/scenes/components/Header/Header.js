import React from 'react'
import styled from 'styled-components'

const Container = styled.section`
  border: 1px solid;
`

const Header = () => (
  <Container>
    <p>Header!</p>
  </Container>
)

export default Header
