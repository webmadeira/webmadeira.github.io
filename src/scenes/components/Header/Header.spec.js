import React from 'react'
import { shallow } from 'enzyme'
import Header from './Header'

describe('Header component', () => {
  it('must render properly', () => {
    const component = shallow(<Header />)

    expect(component).toMatchSnapshot()
  })
})
