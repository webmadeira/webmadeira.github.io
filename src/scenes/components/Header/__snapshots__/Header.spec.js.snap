// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component must render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <p>
        Header!
      </p>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Header!",
      },
      "ref": null,
      "rendered": "Header!",
      "type": "p",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <p>
          Header!
        </p>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Header!",
        },
        "ref": null,
        "rendered": "Header!",
        "type": "p",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
